crates/nostr-sdk/src/client/mod.rs

```diff
@@ -203,8 +203,15 @@ impl Client {
     }
 
     /// Get current [`Keys`]
-    pub fn keys(&self) -> Keys {
-        self.keys.clone()
+    pub async fn keys(&self) -> Keys {
```

Comment:
Isn't turning a synchronous function into an asynchronous one in the public API a breaking change?